apps/playbox/playbox.yml

```
@@ -0,0 +1,14 @@
+name: Playbox
+description: 'An audio playback system for the live production industry.'
```

[Don't start the description with A or An.](https://github.com/electron/apps/blob/master/contributing.md#submission-guidelines)

```suggestion
description: 'Audio playback system for the live production industry.'
```